src/frontend/src/components/mainWindow.ts

```diff
@@ -0,0 +1,53 @@
+import { html, TemplateResult } from "lit-html";
+import { ifDefined } from "lit/directives/if-defined.js";
+import { icLogo } from "./icons";
+import { footer } from "./footer";
+
+/**
+ * main window template
+ *
+ * To avoid having to repeat the same structure in every page,
+ * we use this component to wrap the content of each page.
+ * This way, we can change the structure of the main window
+ * in one place.
+ *
+ * Before it was just a DIV with a class, but now its a more complex
+ * component that includes the logo, the footer, and the container.
+ *
+ */
+export const mainWindow = ({
+  slot,
+  id,
+  showFooter = true,
+  showLogo = true,
```

Comment:
Should these two really be `true` by default? it looks like more often than not they're explicitly set to `false`